lib/pymedphys/docs/appusers/index.rst

```diff
@@ -0,0 +1,7 @@
+======================
+Overview
+======================
+
+Resources to assist users of the PyMedPhys app.
+
+Since the PyMedPhys app has not yet been released, this section is empty!
```

Comment:
Well... it has been released actually. It is currently bootable by running `pymedphys gui` after installing.

It's just that the binary distribution of the PyMedPhys app has not yet been released.

Reply:
Ah yes, good point!